ReactNativeClient/lib/components/screens/encryption-config.js

```diff
@@ -56,6 +66,22 @@ class EncryptionConfigScreenComponent extends BaseScreenComponent {
 		return shared.checkPasswords(this);
 	}
 
+	handlePasswordShow(id) {
+		const prevState = this.state.passwordHide;
+		const prevValue = prevState.find(obj => obj.id === id)[id];
+		const newState = prevState.map(obj => {
+			if (obj.id == id) {
+				return {
+					id: obj.id,
+					[id]: !prevValue,
+				};
+			} else {
+				return obj;
+			}
+		});
+		this.setState({ passwordHide: newState });
+	}
```

Comment:
Please make sure that only one password is shown at a time.

Reply:
Sir, I used characters because, those are used at many places in mobile like even for tick and cross, characters are used. But If you want I will change the things you suggested and get back to you asap. Thanks.

Reply:
Sir, I have done all the changes you suggested. Please have a look and let me know if I need to change anything else.